### PR TITLE
Move live-tests to airbyte repo

### PR DESCRIPTION
**Pull request by clnoll**
Related: https://github.com/airbytehq/airbyte-platform-internal/pull/12115.

### CODE REVIEW

File: airbyte-ci/connectors/live-tests/README.md [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+/tmp/regression_tests_artifacts
+└── session_1710754231
+    |── report.html
+    ├── command_execution_artifacts
+    │   └── source-orb
+    │       ├── check
+    │       │   ├── dev
+    │       │   │   ├── airbyte_messages
+    │       │   │   │   ├── connection_status.jsonl
+    │       │   │   │   ├── duck.db
```
Comment (alafanechere) [inline airbyte-ci/connectors/live-tests/README.md:170]: Nit: I forgot to update the readme, this file is now under the session dir

**Review by alafanechere — APPROVED** [review]

File: airbyte-ci/connectors/live-tests/README.md (hunk 1 above)
Comment (clnoll) [reply airbyte-ci/connectors/live-tests/README.md]: Updated.

**Review by natikgadzhi — APPROVED** [review]
I love this! <3 

Want to add a top-level `airbyte-ci/README.md` entry about live tests as well?

**Comment by clnoll** [comment]
@natikgadzhi once the tests can be run with the `airbyte-ci` command I'll add it to the README!

**Pull request merged**